fusion.ml

```diff
@@ -75,6 +75,7 @@ module type Hol_kernel =
       val variant : term list -> term -> term
       val vsubst : (term * term) list -> term -> term
       val qsubst : (term * term) list -> term -> term
+      val BETA_RED_BY_SUB : term -> thm 
```

Comment:
This stands out (all caps) amongst things that aren't. Does it belong in this part of the signature?